changelog/1195.trivial.rst

```diff
@@ -0,0 +1 @@
+Added `tox` and removed pytest as extra requirement.
```

Comment:
```suggestion
Added `tox <https://tox.readthedocs.io/en/latest/>`_ and removed `pytest <https://docs.pytest.org/>`_ as extra requirement.
```

Looks like `tox`'s documentation is not fully indexed so `tox` is not cross-referenced.  I guess just add the link manually.